include/dsCompositeIn.h

```diff
@@ -355,6 +371,29 @@ dsError_t dsCompositeInRegisterSignalChangeCB (dsCompositeInSignalChangeCB_t CBF
 
 dsError_t dsCompositeInRegisterStatusChangeCB (dsCompositeInStatusChangeCB_t CBFunc);
 
+/**
+ * @brief Registers a callback for the Composite input video mode Change event
+ *
+ * This function registers a callback for the Composite input video mode Change event.
+ *       The mode change is triggered whenever the video resolution changes.
+ *
+ * @param[in] CBFunc    - HDMI input video mode change callback function.
+ *                              Please refer ::dsCompositeInVideoModeUpdateCB_t
+ *
+ * @return dsError_t                        - Status
+ * @retval dsERR_NONE                       - Success
+ * @retval dsERR_NOT_INITIALIZED            - Module is not initialised
+ * @retval dsERR_INVALID_PARAM              - Parameter passed to this function is invalid
+ * @retval dsERR_OPERATION_NOT_SUPPORTED    - The attempted operation is not supported; e.g: source devices
```

Comment:
yes, this can be supported on a panel that doesn't support composite in.  The view of the above comment is that why should source device have this file as it doesn't support a composite In. We can set the example as a "Panel without Composite IN" rather than "source devices"

Reply:
Please consider this at all the locations.